### DISCUSSION TITLE
Display image below text

### DISCUSSION THREAD

**Discussion by NadineDijkstra** [jspsych/jsPsych#740] [Q&A] [answered]
Heya, I want to display an image below text. I am using the "imagine-keyboard-response" plugin but this automatically puts the image above the text. Do I need to try and make my own function to show the image below the text or is there a different way? Thanks!

**Comment by jodeleeuw** [comment, marked as answer]
You can use the `html-keyboard-response` plugin for more customization.

```js
var sample_trial = {
	type: 'html-keyboard-response',
	stimulus: '<p>Text above the image</p>'+
		'<img src="path/to/image.png"></img>'
}
```
